plugins/plugin-site/src/components/ThankAContributorNote.jsx

```diff
@@ -0,0 +1,150 @@
+/* eslint-disable react/jsx-one-expression-per-line */
+
+import React, {useEffect, useState} from 'react';
+import axios from 'axios';
+import dayjs from 'dayjs';
+import {Box, Stack, useTheme} from '@mui/material';
+import Papa from 'papaparse';
+import {useMediaQuery} from '../hooks/useMediaQuery';
+
+function ThankAContributorNote() {
+    const theme = useTheme();
+    const isMobile = useMediaQuery('man-width: 768px)');
+    const isDesktop = useMediaQuery('(min-width: 1024px)');
+
+    const [thankYou, setThankYou] = useState([]);
+
+    useEffect(() => {
+        axios
+            .get(
+                'https://raw.githubusercontent.com/jenkins-infra/jenkins-contribution-stats/main/data/honored_contributor.csv',
+                {responseType: 'text'}
+            )
+            .then((response) => {
+                setThankYou(Papa.parse(response.data)?.data[1]);
+            });
+
+        const interval = setInterval(() => {
+            axios
+                .get(
+                    'https://raw.githubusercontent.com/jenkins-infra/jenkins-contribution-stats/main/data/honored_contributor.csv',
+                    {responseType: 'text'}
+                )
+                .then((response) => {
+                    setThankYou(Papa.parse(response.data)?.data[1]);
+                });
+        }, 3600000);
+
+        return () => clearInterval(interval);
+    }, []);
+
+
+    return (
+        <Box
+            sx={{
+                padding: theme.spacing(5, 2),
+                display: 'flex',
+                justifyContent: 'center',
+                alignItems: 'center',
+            }}
+        >
+            <Box
+                sx={{
+                    padding: isMobile ? theme.spacing(2) : theme.spacing(3),
```

Comment:
It would be more consistent to extract these into a CSS file, then the media queries can be done in CSS, jsx would only set class names.

Reply:
Done